src/scss/components/_badges.scss

```diff
@@ -82,6 +83,10 @@ $badge-rounded-radius       : 14px !default;
             @include font-size(base--scaleUp);
         }
 
+        .c-badge--noPad {
+            padding: 0;
```

Comment:
What does the badge look like without padding?  Just tried it in the component library and it looks pretty strange without any padding at all (know this is a design question, but it's more to do with consistency across SERP & Menu usage)

Reply:
This is for removing the padding of the badges with the span divider:

<img width="125" alt="screen shot 2018-09-05 at 09 59 47" src="https://user-images.githubusercontent.com/5295718/45082978-7f5a5200-b0f2-11e8-85a5-ea595adbc3e2.png">
<img width="106" alt="screen shot 2018-09-05 at 09 59 51" src="https://user-images.githubusercontent.com/5295718/45082979-7f5a5200-b0f2-11e8-8deb-ee0b5906794f.png">

Reply:
Ah, ok – could you make sure this modifier gets added to the badge page in the docs too?

Reply:
no problem!

Reply:
https://github.com/justeat/global-component-library/pull/48